dir.props

```diff
@@ -452,7 +452,7 @@
     <MSBuildPassThroughPropertyList Condition="'$(OutputRid)' != ''">$(MSBuildPassThroughPropertyList) /p:OutputRid=$(OutputRid)</MSBuildPassThroughPropertyList>
   </PropertyGroup>
 
-  <!-- Use Roslyn Compilers to build, if we are not using the .NET SDK to build -->
-  <Import Project="$(RoslynPropsFile)" Condition="'$(UsingNETSdkCompiler)' != 'true' and '$(OsEnvironment)'!='Windows_NT' and Exists('$(RoslynPropsFile)') and '$(UseRoslynCompilers)'!='false'" />
-  <Import Project="$(RoslynPropsFile)" Condition="'$(UsingNETSdkCompiler)' != 'true' and '$(OsEnvironment)'=='Windows_NT' and Exists('$(RoslynPropsFile)')" />
+  <!-- Use Roslyn Compilers to build -->
+  <Import Project="$(RoslynPropsFile)" Condition="'$(RunningOnUnix)'!='true' and Exists('$(RoslynPropsFile)') and '$(UseRoslynCompilers)'!='false' and '$(RoslynIncompatibleMsbuildVersion)' != 'true'" />
+  <Import Project="$(RoslynPropsFile)" Condition="'$(RunningOnUnix)'=='true' and Exists('$(RoslynPropsFile)') and '$(RoslynIncompatibleMsbuildVersion)' != 'true'" />
```

Comment:
Are we planning to do away with this import now we are moving to the CLI msbuild? I wouldn't expect we need this anylonger.

Reply:
we will eventually throw it once we also build using the CLI on Windows, but we still need it for now.

Reply:
We still need it even for the RunningOnUnix == true condition?

Reply:
Yes, because the RoslynPropsFile is set to the props file inside the microsoft.netcore.compilers package. Roslyn.Common.props will set the RoslynPropsFile accordingly depending on if you are running on unix or on windows.